scripts/apps/publish/directives/SubscribersDirective.ts

```diff
@@ -350,32 +350,32 @@ export function SubscribersDirective(
 
                 const {start_date, end_date} = $scope.subscriber.schedule;
                 const now = startOfDay(new Date());
-                const mStart = start_date ? startOfDay(new Date(start_date)) : null;
-                const mEnd = end_date ? startOfDay(new Date(end_date)) : null;
+                const start = start_date ? startOfDay(new Date(start_date)) : null;
+                const end = end_date ? startOfDay(new Date(end_date)) : null;
 
                 if ($scope.subscriber.is_active) {
-                    if (mStart != null && isAfter(mStart, now)) {
+                    if (start != null && isAfter(start, now)) {
```

Comment:
with native dates you can use standard comparison operators, like

`if (start != null && start > now) {`

I'm not requesting you change it, just letting you know for the future.

Reply:
This is noted Tomas. Thanks